src/Common/Core/Impl/Imaging/IImageService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Imaging {
+    /// <summary>
+    /// Provides access to images and icons
+    /// </summary>
+    public interface IImageService {
```

Comment:
Yes, anywhere icons are needed. Which is anywhere in editor for example. BTW, same happens to Components. In plots there won't be 'BitmapSource' anymore, just object.